contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/pycodestyle_subsystem.py

```diff
@@ -46,12 +46,12 @@ class PEP8Subsystem(PluginSubsystemBase):
 
   @classmethod
   def register_options(cls, register):
-    super(PEP8Subsystem, cls).register_options(register)
+    super(PyCodeStyleSubsystem, cls).register_options(register)
     register('--ignore', fingerprint=True, type=list, default=cls.DEFAULT_IGNORE_CODES,
              help='Prevent test failure but still produce output for problems.')
     register('--max-length', fingerprint=True, type=int, default=100,
-             help='Max line length to use for PEP8 checks.')
+             help='Max line length to use for pycodestyle checks.')
 
   def get_plugin_type(self):
-    from pants.contrib.python.checks.tasks.checkstyle.pep8 import PEP8Checker
-    return PEP8Checker
+    from pants.contrib.python.checks.tasks.checkstyle.pycodestyle import PyCodeStyleChecker
```

Comment:
would it make sense to move this import to the top of the file while you're in here?

Reply:
It looks like the late import is done for `get_plugin_type` in every other  `*_subsystem.py` in `contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle`  also.  I'm not sure why. Maybe move all of them in another commit?

Reply:
Please don't. These are imported late for performance reasons.